packages/pug/test/pug.test.js

```diff
@@ -979,12 +979,20 @@ describe('pug', function(){
       assert(actual === expected);
     });
     it('should support .pug.compileClient(str, options)', function () {
-      var src = '.bar= self.foo'
+      var src = '.bar= self.foo';
       var fn = pug.compileClient(src, {self: true});
       fn = Function('pug', fn.toString() + '\nreturn template;')(pug.runtime);
       var actual = fn({foo: 'baz'});
       assert(actual === '<div class="bar">baz</div>');
     });
+    it('should support module syntax .pug.compileClient(str, options)', function () {
+      var src = '.bar= self.foo';
+      var fn = pug.compileClient(src, {self: true, module: true, inlineRuntimeFunctions: false});
```

Comment:
It would be good to have an example with both `inlineRuntimeFunctions: true` and `inlineRuntimeFunctions: false`.

You might also want to add `expect(fn).toMatchSnapshot();` which will make the generated code part of the test, making it easier to see if this functionality gets broken in some subtle way.